packages/polaris-typeorm/src/utils/query-builder-util.ts

```diff
@@ -0,0 +1,46 @@
+import { EntityFilter } from '@enigmatis/polaris-common';
+import { SelectQueryBuilder } from 'typeorm';
+
+export const addDateRangeCriteria = (
+    qb: SelectQueryBuilder<any>,
+    dateRangeFilter: EntityFilter,
+    entityTableName: string,
+) => {
+    if (dateRangeFilter.creationTimeFilter?.gt) {
+        qb.andWhere(`${entityTableName}.creationTime > :gt`, {
+            gt: dateRangeFilter.creationTimeFilter?.gt,
+        });
+    } else if (dateRangeFilter.creationTimeFilter?.gte) {
+        qb.andWhere(`${entityTableName}.creationTime >= :gte`, {
+            gte: dateRangeFilter.creationTimeFilter?.gte,
+        });
+    }
+    if (dateRangeFilter.creationTimeFilter?.lt) {
```

Comment:
put first 2 ifs elses in function, to work once on creation time and once on last update time